src/APICore.ts

```diff
@@ -17,7 +17,7 @@ export default class API {
     }
 
     get organizationId(): string {
-        return retrieve(this.config.organizationId);
+        return retrieve(this.config.organizationId!); // Assuming that resources calling this method would have a defined organizationId
```

Comment:
This is a side effect of #233.
Avoiding refactors right now and keeping the same behaviour.